docs/gke/configs/cluster.jinja

```diff
@@ -28,14 +28,21 @@ resources:
       name: {{ CLUSTER_NAME }}
       # 
       initialNodeCount: {{ properties['initialNodeCount'] }}
-      # We need 1.9 to support GPUs without requiring Alpha.
-      initialClusterVersion: 1.9.6-gke.1
+      # TODO: remove alpha when 10.2 is public.
```

Comment:
Can you file an issue an reference the issue here? Could we add an option that can be enabled via the YAMl spec to enable Alpha features or not? And then selectively enable the logging and monitoring service below?